### ISSUE TITLE
Different inference results with Keras and EDDL

### ISSUE THREAD

**Issue by drodagu** [deephealthproject/eddl#319]
Hi,
I have been doing some tests with a very simple model for regression: 3 dense layers 50 neurons each, 8 inputs and a single output. When I do inference with the same model using Keras and EDDL, the results obtained are different. I'm using  **EDDL v1.0.3b with CUDNN**.

In all the test, the model was initialized in Keras and imported into the EDDL as an ONNX file. The blue dots are the inference results obtained with Keras, while the red dots are the inference results obtained with EDDL.

In the first test, the inference results of Keras were obtained with the model trained using Keras, while for the inference results in EDDL the model was trained in EDDL.

![keras_eddl](https://user-images.githubusercontent.com/80259590/144596228-018d8c1e-57b9-4896-bda9-d69730aa24a5.png)

In the second test, the model was trained with keras.

![keras_trained_eddl](https://user-images.githubusercontent.com/80259590/144596391-07976e59-06d3-4290-ae8c-c624e452d818.png)

And in the last test, the model was trained with EDDL.

![keras_eddl_trained](https://user-images.githubusercontent.com/80259590/144596545-be35b901-dc85-4210-a8f5-666f04023eb8.png)

The Y axis correspond to the expected results (Test_Y) and the X axis correspond to the predicted results. In the three test, EDDL has trouble predicting values that are greater than 0.4. 

The files used to perform the tests are these: [inference_keras_eddl.zip](https://github.com/deephealthproject/eddl/files/7649460/inference_keras_eddl.zip)
